frontend/public/declarations.d.ts

```diff
@@ -45,7 +45,10 @@ declare interface Window {
   __REDUX_DEVTOOLS_EXTENSION_COMPOSE__?: Function;
   store?: {}; // Redux store
   pluginStore?: {}; // Console plugin store
-  Cypress?: {};
+  Cypress?: {
+    numberA11yViolations: number;
+    numberA11yChecks: number;
```

Comment:
If we're only accessing these in the Cypress tests, it would be better to avoid adding the types here since this `window` definition applies to console itself.